tripy/tests/utils/test_utils.py

```diff
@@ -109,3 +110,45 @@ def test_gen_uid(self, inputs, outputs, expected_prefix):
     def test_uniqueness(self):
         uids = [utils.UniqueNameGen.gen_uid() for _ in range(100)]
         assert len(set(uids)) == 100
+
+
+class TestStride:
+    @pytest.mark.parametrize(
+        "shape, expected_stride",
+        [
+            ((), ()),
+            ((1,), (1,)),
+            ((2, 3), (3, 1)),
+            ((0, 5), (1, 1)),
+            ((1, 0, 3), (1, 1, 1)),
+            ((2, 1, 4), (4, 1, 1)),
+            ((3, 0, 0, 5), (5, 1, 1, 1)),
+            ((2, 3), (3, 1)),
+            ((0, 5), (1, 1)),
+            ((2, 0, 3), (3, 1, 1)),
+            (None, None),
+        ],
+    )
+    def test_get_stride(
+        self,
+        shape: Optional[Sequence[int]],
+        expected_stride: Optional[Sequence[int]],
+    ):
+        if shape is not None:
+            assert utils.get_canonical_stride(shape) == expected_stride
```

Comment:
Looks like this test does nothing if `shape` is `None`. Can we remove those cases and remove this check?